notebooks/HASP/DataDiagnostic/DataDiagnostics.ipynb

```diff
@@ -0,0 +1,1599 @@
+{
```

Comment:
<p>I would suggest defining a variable for the visit id, <code>visit_id_to_plot='51'</code> at the beginning of the code so you can easily change which visit ID's data you want to plot without modifying the logic of your code.  Also in the for loop you would also need to update the condition as such:<code> if visit_id == visit_id_to_plot</code>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/207/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>